src/ExpressionEvaluator/Core/Source/Concord/Microsoft.VisualStudio.Debugger.Engine.NetFX20.il

```diff
@@ -11716,7 +11716,7 @@
 		{
 		}
 		.method public virtual hidebysig newslot abstract 
-			instance string 'GetTypeName'(class 'Microsoft.VisualStudio.Debugger.Evaluation'.'DkmInspectionContext' 'inspectionContext', class 'Microsoft.VisualStudio.Debugger.Clr'.'DkmClrType' 'clrType', class ['mscorlib']'System.Collections.ObjectModel'.'ReadOnlyCollection`1'<string> 'formatSpecifiers')
+			instance string 'GetTypeName'(class 'Microsoft.VisualStudio.Debugger.Evaluation'.'DkmInspectionContext' 'inspectionContext', class 'Microsoft.VisualStudio.Debugger.Clr'.'DkmClrType' 'clrType', object clrTypeInfo, class ['mscorlib']'System.Collections.ObjectModel'.'ReadOnlyCollection`1'<string> 'formatSpecifiers')
```

Comment:
I assume these changes go away if you fetch/rebase?  (I would assume the API changes are already in master)